R/check-inputs-scoring-functions.R

```diff
@@ -193,6 +193,57 @@ check_input_binary <- function(observed, predicted) {
 }
 
 
+#' @title Assert that inputs are correct for nominal forecasts
+#' @description Function assesses whether the inputs correspond to the
+#' requirements for scoring nominal forecasts.
+#' @param observed XXX
```

Comment:
Docs are expected to be inherited here or otherwise missed?

Reply:
This is the only point that I think needs to be addressed in this PR

Reply:
Updated the documentation